package.json

```diff
@@ -64,6 +64,6 @@
     "semver": "^6.3.0"
   },
   "peerDependencies": {
-    "bpmn-js": "^3.x || ^4.x || ^5.x || ^6.x || ^7.x || ^8.x"
```

Comment:
Why this change? What actually broke in terms of compatibility?

Reply:
Corrected